### PR TITLE
fix(broker) : update broker retry interval default value and description (21.10)

### PR DESCRIPTION
**Pull request by AssanDiomande**
## Description

update broker retry interval database (add update file for branch 21.10)

**Fixes** MON-4949

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [x] 21.10.x
- [ ] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

* After creating a new poller check in broker configuration if the circular help button mention the 30 sec for retry interval default value

### CODE REVIEW

File: www/install/php/Update-21.10.3.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+/**
+ * Query with transaction
+ */
+try {
+    $pearDB->beginTransaction();
+    $errorMessage = 'Unable to delete logger entry in cb_tag';
+    $statement = $pearDB->query("DELETE FROM cb_tag WHERE tagname = 'logger'");
+    $errorMessage = 'Unable to update the description in cb_field';
+    $statement = $pearDB->query("UPDATE cb_field SET `description` =
+    'Time in seconds to wait between each connection attempt. The default value is 30s.' WHERE `cb_field_id` = 31");
```
Comment (kduret) [inline www/install/php/Update-21.10.3.php:38]: This make the query more readable
```suggestion
    $statement = $pearDB->query("
        UPDATE cb_field
        SET `description` = 'Time in seconds to wait between each connection attempt. The default value is 30s.'
        WHERE `cb_field_id` = 31
    ");
```

File: lang/es_ES.UTF-8/LC_MESSAGES/help.po [hunk 2]
```diff
@@ -213,8 +213,8 @@ msgid "Time in seconds to wait before launching failover."
 msgstr "Tiempo en segundos para esperar antes de habilitar la conmutación por error."
 
 #: centreon-web/help.php:50 centreon-web/www/include/configuration/configCentreonBroker/help.php:88
-msgid "Time in seconds to wait between each connection attempt."
-msgstr "Tiempo de espera en segundos entre cada intento de conexión."
+msgid "Time in seconds to wait between each connection attempt. The default value is 30s."
```
Comment (kduret) [inline lang/es_ES.UTF-8/LC_MESSAGES/help.po:216]: Can you change this everywhere ?
This make also the translation consistent
```suggestion
msgid "Time in seconds to wait between each connection attempt (Default value: 30s)."
```

**Review by a-launois — APPROVED** [review]

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 3]
```diff
@@ -303,8 +303,8 @@ msgstr "Temps en secondes à attendre avant d'activer le basculement."
 
 #: centreon-web/help.php:50
 #: centreon-web/www/include/configuration/configCentreonBroker/help.php:88
-msgid "Time in seconds to wait between each connection attempt."
-msgstr "Temps d'attente en secondes entre chaque tentative de connexion."
+msgid "Time in seconds to wait between each connection attempt (Default value: 30s)."
+msgstr "Temps d'attente en secondes entre chaque tentative de connexion (par défaut 30s)."
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:307]: it's really important to be consistent between translations
```suggestion
msgstr "Temps d'attente en secondes entre chaque tentative de connexion (Valeur par défaut: 30s)."
```

File: www/install/insertBaseConf.sql [hunk 4]
```diff
@@ -589,7 +589,7 @@ INSERT INTO `cb_field` (`cb_field_id`, `fieldname`, `displayname`, `description`
 (26, 'compression_level', 'Compression level', 'Ranges from 0 (no compression) to 9 (best compression). Default is -1 (zlib compression)', 'int', NULL),
 (27, 'compression_buffer', 'Compression buffer size', 'The higher the buffer size is, the best compression. This however increase data streaming latency. Use with caution.', 'int', NULL),
 (28, 'failover', 'Failover name', 'Name of the input or output object that will act as failover.', 'text', NULL),
-(31, 'retry_interval', 'Retry interval', 'Time in seconds to wait between each connection attempt.', 'int', NULL),
+(31, 'retry_interval', 'Retry interval', 'Time in seconds to wait between each connection attempt. The default value is 30s.', 'int', NULL),
```
Comment (kduret) [inline www/install/insertBaseConf.sql:592]:
```suggestion
(31, 'retry_interval', 'Retry interval', 'Time in seconds to wait between each connection attempt (Default value: 30s).', 'int', NULL),
```

File: www/include/configuration/configCentreonBroker/help.php [hunk 5]
```diff
@@ -83,7 +83,7 @@
 echo _("The maximum queries per transaction before commit.");
 echo _("The transaction timeout before running commit.");
 echo _("The interval between check if some metrics must be rebuild. The default value is 300s");
-echo _("Time in seconds to wait between each connection attempt.");
+echo _("Time in seconds to wait between each connection attempt. The default value is 30s.");
```
Comment (kduret) [inline www/include/configuration/configCentreonBroker/help.php:86]:
```suggestion
echo _("Time in seconds to wait between each connection attempt (Default value: 30s).");
```

File: www/install/php/Update-21.10.3.php [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+ * Query with transaction
+ */
+try {
+    $pearDB->beginTransaction();
+    $errorMessage = 'Unable to delete logger entry in cb_tag';
+    $statement = $pearDB->query("DELETE FROM cb_tag WHERE tagname = 'logger'");
+    $errorMessage = 'Unable to update the description in cb_field';
+    $statement = $pearDB->query("
+        UPDATE cb_field
+        SET `description` = 'Time in seconds to wait between each connection attempt. The default value is 30s.'
```
Comment (kduret) [inline www/install/php/Update-21.10.3.php:39]:
```suggestion
        SET `description` = 'Time in seconds to wait between each connection attempt (Default value: 30s).'
```

**Review by kduret — APPROVED** [review]

**Pull request merged**